drivers/wifi/winc1500/wifi_winc1500_nm_bsp_internal.h

```diff
@@ -23,6 +23,19 @@ extern tstrNmBusCapabilities egstrNmBusCapabilities;
 #define NM_DEBUG		CONF_WINC_DEBUG
 #define NM_BSP_PRINTF		CONF_WINC_PRINTF
 
+enum winc1500_gpio_index {
+	WINC1500_GPIO_IDX_CHIP_EN = 0,
+	WINC1500_GPIO_IDX_IRQN,
+	WINC1500_GPIO_IDX_RESET_N,
```

Comment:
In case of the reset pin we generally let the user configure its active level in DTS. I propose to follow the convention. In this case the name of the pin should reflect signal logical value. I.e. it should be `WINC1500_GPIO_IDX_RESET`. Then in the code we would call `gpio_pin_set(..., 1)` to assert the reset (e.g. pull reset pin low) and `gpio_pin_set(..., 0)` to de-assert the reset.

Reply:
No need to change that. Here the point of that PR is just to switch to new API, not to do more work.

Reply:
I agree this rework is not necessary given the scope of this work.  We have not enforced the convert-to-logic-API in every driver that's already been accepted as converted.

Reply:
If we prefer to stick to the old implementation - do not use logical levels - then we should use the `gpio_pin_set_raw` functions. Otherwise the implementation is inconsistent, also quite confusing to the users reading/using the code.

Reply:
I agree, this like #22241 should be using the raw API if it's not going to be updated to use the logical values.